### PR TITLE
RAIL-3720 Update link to setting up backend and workspace

### PR DESCRIPTION
**Pull request by plavemar**
JIRA: RAIL-3720

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/845/ : SUCCESS in 1m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/554/ : SUCCESS in 6s

File: docs/10_vis__theme_provider.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ copyright: (C) 2007-2020 GoodData Corporation
 id: theme_provider
 ---
 
-The **Theme Provider component** allows you to customize the visual style of your dashboards by applying a theme (see [Create Custom Themes](https://help.gooddata.com/pages/viewpage.action?pageId=81974323)).
+The **Theme Provider component** allows you to customize the visual style of your dashboards by applying a theme (see [Create Custom Themes](https://help.gooddata.com/pages/viewpage.action?pageId=86794481)).
 
-To be able to use the Theme Provider component, set up the `backend` and `workspace` properties in your application (see [Connecting to an Analytical Backend](02_start__connecting_backend.md#initial-setup)).
+To be able to use the Theme Provider component, set up the `backend` and `workspace` properties in your application (see [Connecting to an Analytical Backend](06_cloudnative__integration.md#Step-3.-Set-up-Analytical-Backend-and-integrate-it-into-your-application)).
```
Comment (no23reason) [inline docs/10_vis__theme_provider.md:10]: Previously the linked page was for a bear version, so it should probably be
`02_start__no_boilerplate.md#step-3-set-up-analytical-backend-and-integrate-it-into-your-application` now, right?
Or maybe we should expand it to consider both bear and tiger?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/848/ : SUCCESS in 1m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/556/ : SUCCESS in 10s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR452

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/849/ : SUCCESS in 1m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/154/ : SUCCESS in 43s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/52/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/242/ : SUCCESS in 18s

**Pull request merged**